init/systemd/ipa.sysusers.base.conf

```diff
@@ -0,0 +1,5 @@
+u kdcproxy  -   "IPA KDC Proxy User"    -
+u dirsrv    -   -   -
```

Comment:
+1 for Honza's suggestion. Let 389-ds-base take care of group and user creation.